File: mario.c
#include <cs50.h>
#include <stdio.h>

int get_positive_int(void);

int main(void)
{
   int n;
    do
    {
        n = get_positive_int();
    }
    while (n < 1 || n > 8);
       
    for (int rows = 1; rows <= n; rows++)
    {
        for (int dots = 0; dots < n-rows; dots++)
        {
            printf(" ");
        }
        for (int hashtag = 0; hashtag < rows; hashtag++)
        {
            printf("#");
        }
         printf("\n");
        }
       
    }
        


int get_positive_int(void)
{
    int n;
    do
    {
        n = get_int("Height: ");
    }
    while (n < 1);
    return n;
   
}

    

File: mario.py
from cs50 import get_int

# Prompt user for number between 1 and 8
while True:
    n = get_int("Height: ")
    if n >= 1 and n <= 8:
        break

r = 1
d = 0
h = 0

# go through a loop and write out hashes and spaces
for r in range(n):
    print(" " * (n - r - 1), "#" * (r + 1))

